plugins/source/cloudflare/docs/index.md

```diff
@@ -1,54 +0,0 @@
-## Cloudflare Plugin
-
-The CloudQuery Cloudflare provider pulls configuration out of Cloudflare resources, normalizes them and stores them in PostgreSQL database.
-
-### Install
-
-```bash
-cloudquery init cloudflare
```

Comment:
This will need to be updated (`cloudquery generate cloudflare`) here and lower down. It would also be good to add a sentence or two about what the command does: it generates a file called `cloudflare.yml` with example config that needs to be tweaked.

Reply:
Updated